### ISSUE TITLE
NullPointerException guarding

### ISSUE THREAD

**Issue by kylegrymonprez** [Swrve/swrve-android-sdk#262]
We're seeing reports of a NullPointerException at `com.swrve.sdk.gcm.SwrveGcmIntentService.onCreate(SwrveGcmIntentService.java:28)`

Could you please guard that line with a null check?
```
    @Override
    public void onCreate() {
        super.onCreate();
        pushSDK = SwrvePushSDK.getInstance();
        **if (pushSDK != null)**
           pushSDK.setService(this);
}
```

Thanks!

**Comment by Sergio-Mira** [comment]
Hi @wacamoe thanks for giving us the heads up. We'll look into it and fix it, have a nice weekend!

**Comment by Sergio-Mira** [comment]
Is the SDK initialised on Application:onCreate? If that is the case then there should always be a SwrvePushSDK instance. If there isn't one then we won't be able to display the notification received from the Swrve backend.

**Comment by rmccormickglu** [comment]
@Sergio-Mira It was not.  We were initializing in Activity:onCreate, which seemed to work just fine on  the previous version we were using - I believe it was 4.6.1.

I'll see if this fixes our issue. Thank you!

**Comment by Sergio-Mira** [comment]
We've released 4.10 which includes guards in case this happens for some reason, let us know if you are still having issues and reopen the ticket, thanks! 👍
